ui/pages/home/home.component.js

```diff
@@ -1,6 +1,7 @@
 import React, { PureComponent } from 'react';
```

Comment:
This page could use a much larger refactor but in this PR we are just replacing the Typography component.

Reply:
`PureComponent` 👀